### PR TITLE
Create categories.yaml

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

This file contains info for every category we intend to have for the integrations on the Data collection parent category.

Currently no description of the category itself was in the related [JSON](https://github.com/netdata/website/blob/master/data/integrations.json), so I left it empty.

All of the categories are under the `data-collection` key, so we can extend this in the future.

All of the categories have most-popular as false, we can manually make categories popular by switching the boolean value as we see fit.

id is essentially the name from the JSON, with all the spaces and special characters replaced by a dash.

### CODE REVIEW

File: integrations/categories.yaml [hunk 1]
```diff
@@ -0,0 +1,251 @@
+data-collection:
+  - id: containers-and-vms
+    parent: data-collection
+    name: Containers and VMs
+    description: ""
+    most-popular: false
```
Comment (ilyam8) [inline integrations/categories.yaml:6]: How did you generate this list? Any reason to not have it nested/without "parent"? This is extremely confusing as it is.

Comment (Ancairon) [reply integrations/categories.yaml:6]: I talked the exact same with Austin, we decided it would need to be in the final data regardless, and handling it from the beginning like this is easier for Austin.

As far as how this was made, I got the childTag from the JSON from each entry, kept the unique ones, and then generated the rest.

Essentially I only had `name` from JSON, and made `id` from that.

Comment (ilyam8) [reply integrations/categories.yaml:6]: I don't think it is easier or harder. @Ferroin is going to be using python and it is super easy to convert to whatever data structure he wants during processing.

Comment (ilyam8) [reply integrations/categories.yaml:6]: Also, I see that the "parent" is always `data-collection` and the top-level key is `data-collection`. Why do we need "parent"? Is it possible to have different values?

Comment (Ancairon) [reply integrations/categories.yaml:6]: I mean, I don't have a preference, that's why I asked Austin if he had one, sure we can do this in a million ways, and still be correct

Comment (Ancairon) [reply integrations/categories.yaml:6]: > Also, I see that the "parent" is always data-collection and the top-level key is data-collection. Why do we need "parent"? Is it possible to have different values?

That's what I discussed with Austin, he said we should leave it in anyways

Comment (Ferroin) [reply integrations/categories.yaml:6]: The structure here is mirroring what the frontend expects, with the intent being getting things working quickly. The schema can be found here: https://github.com/netdata/netdata/blob/master/docs/.templates/integration/schema.json, this file is supposed to mirror the ‘categories’ property of the structure described there.

If we want to use a nested structure with something like a `children` key we can look into doing that eventually, but

**Comment by Ancairon** [comment]
I added the deploy category, with the categories indicated by [this sheet](https://docs.google.com/spreadsheets/d/1llxBXkgv3Z1Rsgeeua1eZB6LSkCVwJCHzyUoecIspQ8/edit#gid=0).

File: integrations/categories.yaml (hunk 1 above)
Comment (ilyam8) [reply integrations/categories.yaml:22]: But it doesn't mirror (e.g. has `most-popular`, `parent` instead of `parent_id`). I just found it strange to have `X` as a top-level key and `X` as a parent.

File: integrations/categories.yaml [hunk 2]
```diff
@@ -0,0 +1,267 @@
+deploy:
+  - id: operating-systems
+    parent: deploy
+    name: Operating Systems
+    description: ""
+    most-popular: false
+  - id: docker-kubernetes
+    parent: deploy
+    name: Docker & Kubernetes
+    description: ""
```
Comment (ilyam8) [inline integrations/categories.yaml:10]: I think we need to separate categories because Docker and K8s have different deploy instructions.

Comment (Ancairon) [reply integrations/categories.yaml:10]: that's another thing that is grey on testing, shouldn't k8s and docker be cards inside that category? (at the moment that category is empty, was told to add those categories)

Comment (Ancairon) [reply integrations/categories.yaml:10]: Because then you would have only one card 
image

for k8s inside Kubernetes

Comment (ilyam8) [reply integrations/categories.yaml:10]: > then you would have only one card

You are right!

**Comment by Ancairon** [comment]
@Ferroin @kapantzak This should be what we need for categories, the recursion in `docs/.templates/integration/schema.json` is defined as stated in [this source](https://json-schema.org/understanding-json-schema/structuring.html)

**Review by Ferroin — DISMISSED** [review]

**Review by Ferroin — APPROVED** [review]

**Review by hugovalente-pm — APPROVED** [review]
